Add notes on multiline support

d4nyll opened the pull request:
Implement part of #4

d4nyll committed:
Add notes on multiline support

Implement part of #4

AndersDJohnson commented on README.md:
````diff
@@ -22,4 +22,18 @@ Or you can use a custom file name instead of the default `.env`, e.g., `.env.tes
+To export multiline environment variables (as supported in the [`dotenv`](https://github.com/motdotla/dotenv#rules) package), use the literal characters `\n` in your variable:
+
+```
+PUBLIC_KEY="-----BEGIN RSA PUBLIC KEY-----\nMIICCgKCAgEAsTwK1Tireh3TVaJ66yUEAtLPP5tNuqwZW/kA64t7hgIRVKee1Wjb\nKLcHIJcAcioHJnqME96M+YRaj/xvlIFSwIbY1CRPgRkqH7kHs6mnrOIvmiRTPxSO\ntzydJxNB/13RsnuwPpG0MI/ZDNnSgj9sWVtJu4cgwoGpKzZctfhwZzJGBns9wChm\n2HewSTvUSRD2pgpheNS0kFkCsNmZgdhF2qaDfOXS6ie9+nKEms5/Y5/yA0EvAORh\nxnkw/JOkoOBoFX5DaZhQIAuqLmzmWvQqKxC1QTwgZh4zZw5hviBWRzh38uX/hIM7\nLe4lW0jxAIIMiPCcLnj5i9hqE ...
````
Can you please not include what appears to be a real public key in the README? Something less conspicuous with fewer security implications would be preferred.

d4nyll replied:
Done :)

d4nyll committed:
Change to simpler example

Note: The RSA key removed is not used in a real environment, and was created anew to demonstrate a real-world example of this use case.
